.github/workflows.src/build.inc.yml

```diff
@@ -364,6 +364,27 @@
 <%- endfor %>
 <%- endif %>
 
+<% if subdist == "" or subdist == "testing" %>
+  publish-github-release:
+    needs: collect
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Publish Github Release
+      uses: elprans/gh-action-create-release@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: ${{ github.ref }}
+        target: ${{ github.event.pull_request.base.ref }}
+        body: EdgeDB v${{ steps.relver.outputs.version }}
```

Comment:
Should probably include a list of major features and a link to the changelog. Not sure how to automate the feature list, but perhaps just edit manually for now.